lib/rubocop/cop/rspec/focus.rb

```diff
@@ -26,18 +26,16 @@ class Focus < Cop
           ExampleGroups::GROUPS  +
           ExampleGroups::SKIPPED +
           Examples::EXAMPLES     +
-          Examples::SKIPPED
+          Examples::SKIPPED      +
+          Examples::PENDING
 
         focused = ExampleGroups::FOCUSED + Examples::FOCUSED
 
         FOCUSABLE_SELECTORS = focusable.node_pattern_union
 
-        FOCUS_SYMBOL = s(:sym, :focus)
-        FOCUS_TRUE   = s(:pair, FOCUS_SYMBOL, s(:true))
-
         def_node_matcher :metadata, <<-PATTERN
-          {(send #{RSPEC} #{FOCUSABLE_SELECTORS} ... (hash $...))
-           (send #{RSPEC} #{FOCUSABLE_SELECTORS} $...)}
+          {(send #{RSPEC} #{FOCUSABLE_SELECTORS} <$(sym :focus) ...>)
```

Comment:
I believe we need to update the minimum required rubocop version to use this. But nearly every cop that deals with a hash could take advantage of this, so maybe we just need to open a PR with the related updates?

Reply:
Ah, ok. I didn't realize how new it was. It's possible write without using this, but this was definitely the cleanest option I came up with.

Definitely worth reviewing other hash matchers for similar edge cases and simplifications.

I can open a PR to update our constraints if you think that's reasonable.

Reply:
I’d love to see a PR that updates our Rubocop version requirement, and uses the new node matcher syntax where appropriate.

Reply:
#804 does it 🎉